### ISSUE TITLE
About JSON interface

### ISSUE THREAD

**Issue by ghost** [NimParsers/parsetoml#21]
I like it, but it has one big drawback, code, which previously was written like:
```nim
config.getIntArray("admins")
``` 
where config is a section of a TOML file
now needs to be written like:
```nim
import sequtils
config["admins"].getElems().mapIt(it.getNum())
```

**Comment by PMunch** [comment]
Yeah, I was still working on it. And the PR was just to show what I was doing. Not sure why @data-man merged it but we should probably undo that until it's done properly.

**Comment by data-man** [comment]
Within the PR we would not receive feedback. :)

**Comment by PMunch** [comment]
How does JSON handle this? Wouldn't it have the same issue?

**Comment by ghost** [comment]
@PMunch yeah, with JSON I would do it the same way, but with older API of this lib it was easier :) But yeah, it's fine for me

**Comment by PMunch** [comment]
I'm considering splitting this into multiple files. One with the object definition and parsing/converting stuff, and others with interface logic. This way we can actually have different interfaces.
